### DISCUSSION TITLE
Unable to update powershell

### DISCUSSION THREAD

**Discussion by sweinst** [ScoopInstaller/Scoop#4904] [Help and Questions] [answered]
I've tried to update PowerShell with the command "scoop update pwsh"
Bit I get the error message **"ERROR Application "pwsh" is still running. Close all instances and try again"**

I guess it's because scoop uses pwsh for the update script. Is there a way around this?

**Comment by rashil2000** [comment, marked as answer]
Use Windows PowerShell to run `scoop update pwsh`

**Comment by sweinst** [comment]
Thanks, it worked
